docs/source/ingest/destination_connectors/data/elasticsearch_elements_mappings.json

```diff
@@ -0,0 +1,144 @@
+{
+    "properties":{
+       "element_id":{
+          "type":"keyword"
+       },
+       "text":{
+          "type":"text",
+          "analyzer":"english"
+       },
+       "type":{
+          "type":"keyword"
+       },
+       "embeddings":{
+          "type":"float"
+       },
+       "metadata":{
+          "type":"object",
+          "properties":{
+             "category_depth":{
+                "type":"integer"
+             },
+             "parent_id":{
+                "type":"keyword"
+             },
+             "attached_to_filename":{
+                "type":"keyword"
+             },
+             "filetype":{
+                "type":"keyword"
+             },
+             "last_modified":{
+                "type":"date"
+             },
+             "file_directory":{
+                "type":"keyword"
+             },
+             "filename":{
+                "type":"keyword"
+             },
+             "data_source":{
+                "type":"object",
+                "properties":{
+                   "url":{
+                      "type":"text",
+                      "analyzer":"standard"
+                   },
+                   "version":{
+                      "type":"keyword"
+                   },
+                   "date_created":{
+                      "type":"date"
+                   },
+                   "date_modified":{
+                      "type":"date"
+                   },
+                   "date_processed":{
+                      "type":"date"
+                   },
+                   "record_locator":{
+                      "type":"keyword"
+                   },
+                   "permissions_data":{
+                      "type":"object"
+                   }
+                }
+             },
+             "coordinates":{
+                "type":"object",
+                "properties":{
+                   "system":{
+                      "type":"keyword"
+                   },
+                   "layout_width":{
+                      "type":"float"
+                   },
+                   "layout_height":{
+                      "type":"float"
+                   },
+                   "points":{
+                      "type":"float"
+                   }
+                }
+             },
+             "languages":{
```

Comment:
This comes in as a list in the json. Is `keyword` correct?

Reply:
Yes, from https://www.elastic.co/guide/en/elasticsearch/reference/current/array.html#array:

> In Elasticsearch, there is no dedicated array data type. Any field can contain zero or more values by default, however, all values in the array must be of the same data type. For instance:
> 
> an array of strings: [ "one", "two" ]
> an array of integers: [ 1, 2 ]
> an array of arrays: [ 1, [ 2, 3 ]] which is the equivalent of [ 1, 2, 3 ]
> an array of objects: [ { "name": "Mary", "age": 12 }, { "name": "John", "age": 10 }]

Reply:
Thanks for the explanation!